docs/css/base.css

```diff
@@ -22,4 +22,23 @@
   --danger-light-color: #f8d7da;
   --warning-light-color: #fff385;
   --info-light-color: #cff4fc;
+
+  --size-2xs: 4px;
```

Comment:
2xs? I didn't understand it as xs=0.8rem and 2xs=4px!!??

Reply:
xs means xtra small and 2xs means xxs(extra extra small)